src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -8,7 +8,7 @@ azure-cli-core==2.53.1
 azure-cli-telemetry==1.1.0
 azure-cli==2.53.1
 azure-common==1.1.22
-azure-core==1.26.0
+azure-core==1.28.0
```

Comment:
cc @bebound for reviewing this change

Reply:
I don't find any breaking change from 1.26.0 to 1.28.0: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/core/azure-core/CHANGELOG.md

But `azure-synapse-artifacts==0.18.0` only requires `azure-core>=1.26.2`?
```
azure-synapse-artifacts==0.18.0           Microsoft Azure Synapse Artifacts Client Library for Python
├── azure-common~=1.1                     Microsoft Azure Client Library for Python (Common)
├── azure-mgmt-core<2.0.0,>=1.3.2         Microsoft Azure Management Core Library for Python
│   └── azure-core<2.0.0,>=1.26.2         Microsoft Azure Core Library for Python
│       ├── requests>=2.18.4              Python HTTP for Humans.
│       │   ├── certifi>=2017.4.17        Python package for providing Mozilla's CA Bundle.
│       │   ├── charset-normalizer<4,>=2  The Real First Universal Charset Detector. Open, modern and actively maintained alternative to Chardet.
│       │   ├── idna<4,>=2.5              Internationalized Domain Names in Applications (IDNA)
│       │   └── urllib3<3,>=1.21.1        HTTP library with thread-safe connection pooling, file post, and more.
│       ├── six>=1.11.0                   Python 2 and 3 compatibility utilities
│       └── typing-extensions>=4.6.0      Backported and Experimental Type Hints for Python 3.8+
└── isodate<1.0.0,>=0.6.1                 An ISO 8601 date/time/duration parser and formatter
    └── six                               Python 2 and 3 compatibility utilities
```

Reply:
`azure-synapse-artifacts==0.18.0` should update it's requirement `azure-core>=1.28`. With `azure-core==1.26`, it will throw exception `module 'azure.core.pipeline.policies' has no attribute 'SensitiveHeaderCleanupPolicy'`